src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -267,6 +268,13 @@ export module DotNet {
      * @param resultOrError The serialized result or the serialized error from the async operation.
      */
     endInvokeJSFromDotNet(callId: number, succeeded: boolean, resultOrError: any): void;
+
+    /**
+     * Receives notification that a byte array is available for transfer.
```

Comment:
Minor detail, but helps with keeping track of which direction these calls are going in, since there are symmetric APIs going the other way too :)

```suggestion
     * Invoked by the runtime to transfer a byte array to .NET.
```